### DISCUSSION TITLE
Toggle button for multiple scenes

### DISCUSSION THREAD

**Discussion by lordhelmet77** [Blackymas/NSPanel_HA_Blueprint#1043] [Ideas]
Something I'd love to see would be a button-type, that toggles through multiple scenes. My primary use case would be scenes for daytimes/activities like:

**button "intensity"**
morning --> noon --> afternoon --> evening    or

**button "mood"**
focus --> reading --> chillout --> movienight

Instead of having a page full of buttons, I'd rather toggle through scenes with one button. 
Or open a subpage with the scenes on. Both would be helpful, but I'd prefer the toggle.

**Comment by edwardtfn** [comment]
You can create scenes in Home Assistant and assign those to your panel's buttons, no?

**Comment by topse** [comment]
How can I highlight the active scene?

**Reply by edwardtfn** [reply]
What do you mean by "highlight a scene"?

**Reply by topse** [reply]
Hi,
Sorry for the brief question. When I have a page with lets say 4 scenes, I want the activated scene to be highlighted, like an activated switch or so...

But I can make the page with 4 Scenes, activate them with a press, but the activated scene is not highlighted.

Is this description better?

Thanks and Sorry and Best Regards,
Tobias

**Reply by edwardtfn** [reply]
The problem is that scenes are stateless entities in Home Assistant. It contains the information about when it was last triggered, but as far as I know (and I have to say I don’t use scenes myself) there’s no way to know its state (as it doesn’t have a state).

**Reply by topse** [reply]
Ahh... OK... that would be a problem then... Then I have to look for a workaround.

Thanks and Best Regards,
Tobias
